src/main/webapp/style.css

```diff
@@ -1,9 +1,3 @@
-#content {
```

Comment:
Is this change deliberate?

Reply:
Yes, the content id is being handled in the global.css file now.